smriprep/interfaces/templateflow.py

```diff
@@ -206,4 +212,6 @@ def fetch_template_files(
     files['mask'] = tf.get(name[0], desc='brain', suffix='mask', **specs) or tf.get(
         name[0], label='brain', suffix='mask', **specs
     )
+    # Not guaranteed to exist so add fallback
+    files['t2w'] = tf.get(name[0], desc=None, suffix='T2w', **specs) or Undefined
```

Comment:
Is it guaranteed to be only one file?

Reply:
nope, and neither are the above 😄  - should i add a check?

Reply:
for the sake of progress, we'll keep an eye out and address accordingly